modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -682,8 +682,9 @@ public class ApiClient {
 
     Entity<?> entity = serialize(body, formParams, contentType);
 
-    Response response;
+    Response response = null;
 
+    try {
     if ("GET".equals(method)) {
```

Comment:
You are right. I have no experience with GitHub, but I will try to do what you mention me